app/scripts/metamask-controller.js

```diff
@@ -4420,12 +4423,30 @@ export default class MetamaskController extends EventEmitter {
           this.permissionController,
           origin,
         ),
+        hasPermission: this.permissionController.hasPermission.bind(
```

Comment:
do these need to be inside `///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)`?

Reply:
We don't use these in any other RPC methods, so yeah.

Reply:
Hum, maybe a bad cherry-pick? 🤔

I'll move it.